src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -74,42 +74,37 @@ public void RawSendInCaseOfFailure(IModel channel, string address, byte[] body,
 
         public void Initialize(IModel channel, string mainQueue)
         {
-
             CreateExchange(channel, mainQueue);
             channel.QueueBind(mainQueue, mainQueue, string.Empty);
-
         }
 
-        static string ExchangeName(Type type)
-        {
-            return type.Namespace + ":" + type.Name;
-        }
+        static string ExchangeName(Type type) => type.Namespace + ":" + type.Name;
```

Comment:
> This already happens automatically

Interesting, I didn't know that! Up to four operands, apparently.

> Is this a more general statement?

Yep, it's a general statement, very much so. Current culture is often (unknowingly) misused and in most cases, it just happens to work because there's nothing that in the formatting that would be affected. In most other projects I work on, we allow code analysis to tell us when we forget to specify the culture.